Tasks/Common/artifacts-common/retryUtils.ts

```diff
@@ -11,7 +11,12 @@ export async function retryOnException<T>(action: () => Promise<T>, maxTries: nu
                 throw error;
             }
             tl.debug(`Attempt failed. Number of tries left: ${maxTries}`);
-            tl.debug(JSON.stringify(error));
+            if (error instanceof Error) {
```

Comment:
Why can't use the logError function here?

Reply:
It's because I put the helper in packaging-common and this is in artifacts-common. I didn't want to create a dependency between the two common folders.